starknet_py/net/base_client_schemas.py

```diff
@@ -0,0 +1,12 @@
+from marshmallow import Schema, fields, post_load
+from starknet_py.net.base_client import BlockHashIdentifier, BlockNumberIdentifier
+
+
+class BlockHashIdentifierSchema(Schema):
+    block_hash = fields.Integer()
+    index = fields.Integer()
+
+
+class BlockNumberIdentifierSchema(Schema):
+    block_number = fields.Integer()
+    index = fields.Integer()
```

Comment:
Why do we need schemas for **input**? I think that if we specify expected type and someone provides invalid value it is their problem.

Reply:
Removed